### ISSUE TITLE
[Question] VecNorms clip and Gym env's observation_space

### ISSUE THREAD

**Issue by VVIERV00** [DLR-RM/stable-baselines3#770]
### Question
When using sb3 VecNormalize do we need to pair clip_obs parameter with spaces.Box's lows and highs? 
Like if you set a box with low=-1, high=1 shouldn't we change the default param in VecNorms to 1? (max absolute value)
Also, Is it correct to guess that lows == 0 should never be used with VecNormalize as it provides negative values between  [-MAX, MAX]?
### Checklist

- [yes] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/) (**required**)
- [yes] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo (**required**)

**Comment by Miffyli** [comment]
No, you do not need to set `clip_obs` to match what you have in your Box space. And no, lows can be anything (but ideally small) and values can be negative, the neural network does not mind. The idea of VecNormalize is that you do not have to fine-tune the parameters of Box space, because VecNormalize will transform observations to normal distribution.

**Comment by VVIERV00** [comment]
I see. So, in case I send to VecNorm values like mean 100 std 10 (no negatives) and VecNorm is left with default parameters, what Box lows and high params are preferable?

**Comment by Miffyli** [comment]
See [this](https://stable-baselines3.readthedocs.io/en/master/guide/rl_tips.html#tips-and-tricks-when-creating-a-custom-environment) for answers :). Feel free to close this issue unless you have bugs to report or enhancement ideas to propose.
